tf_encrypted/config.py

```diff
@@ -139,11 +144,18 @@ def add_player(self, name):
   def players(self):
     return self._players
 
-  def get_player(self, name):
-    player = next(
-        (player for player in self._players if player.name == name), None)
+  def get_player(self, name_or_player):
```

Comment:
should these changes be in a separate commit? or at least in the PR description

Reply:
they're definitely dependencies of the major code changes, but this is new behavior for a public-facing API so should probably be mentioned

Reply:
yeah I thought about it as well, will do that